src/systems/filecoin_blockchain/struct/block/block.id

```diff
@@ -40,11 +39,15 @@ type BlockHeader struct {
     //	ComputeUnsignedFingerprint() []
 }
 
+type TxMeta struct {
+    BLSMessages [msg.UnsignedMessage]
+    SECPMessages [msg.SignedMessage]
+}
+
 // Internal representation of a full block, with all messages.
 type Block struct {
     Header        BlockHeader
-    BLSMessages   [msg.UnsignedMessage]
-    SECPMessages  [msg.SignedMessage]
+    Messages      TxMeta
```

Comment:
I suggest reverting this one back: it's a more convenient internal representation.